EntityLabelLib-Bukkit-NmsImpl/src/main/java/io/github/arcaneplugins/entitylabellib/bukkit/nms/NmsLabelHandler.java

```diff
@@ -88,8 +117,27 @@ public void updateEntityLabel(
     ) {
         Objects.requireNonNull(entity, "entity");
         Objects.requireNonNull(packetRecipient, "packetRecipient");
-        //TODO
-        throw new RuntimeException("Not implemented");
+
+        net.minecraft.world.entity.Entity entityHandle = ((CraftEntity) entity).getHandle();
+        ServerPlayer playerHandle = ((CraftPlayer) packetRecipient).getHandle();
+
+        SynchedEntityData entityData = new SynchedEntityData(entityHandle);
+        entityData.set(LABEL_VISIBLE, labelAlwaysVisible);
+        this.sendEntityData(playerHandle, entityHandle, entityData);
+    }
+
+    private void sendEntityData(
+            ServerPlayer playerHandle,
+            net.minecraft.world.entity.Entity entityHandle,
+            SynchedEntityData entityData) {
+        List<DataValue<?>> list = entityData.packDirty();
+        if (list == null || list.isEmpty()) {
+            return;
+        }
```

Comment:
```suggestion
        if (list == null || list.isEmpty()) return;
```

Reply:
Nah. Not using brackets is against good conventions.